cms/djangoapps/contentstore/tests/test_i18n.py

```diff
@@ -152,9 +153,9 @@ def test_message_catalog_translations(self):
         with mock.patch('gettext.translation', return_value=_translator(domain='text', localedir=localedir,
                                                                         languages=[get_language()])):
             i18n_service = self.get_module_i18n_service(self.descriptor)
-            self.assertEqual(i18n_service.ugettext('Hello'), 'Hello')
-            self.assertNotEqual(i18n_service.ugettext('Hello'), 'fr-hello-world')
-            self.assertNotEqual(i18n_service.ugettext('Hello'), 'es-hello-world')
+            self.assertEqual(edx_six.get_gettext(i18n_service)('Hello'), 'Hello')
```

Comment:
NIT: You don't need to call `edx_six.get_gettext` because there is already an import of just the `get_gettext` function.  So you can remove the `from openedx.core.lib import edx_six` import and update these lines to the for `get_gettext(i18n_service)`

Not required but just makes the code in the file more consistent.